cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,20 +1,28 @@
+import {AddressStep, LoginPage, MenuContentPage, PaymentStep, ProductListPage,
+  ShippingStep, ShoppingCartPage} from "../page/index";
```

Comment:
can you rewrite this?
```suggestion
import {
  AddressStep,
  LoginPage,
  MenuContentPage,
  PaymentStep,
  ProductListPage,
  ShippingStep,
  ShoppingCartPage
} from "../page/index";
```

Reply:
Done